README.md

```diff
@@ -9,23 +9,32 @@ Groupie lets you treat your content as logical groups and handles change notific
 # Try it out:
 
 ```gradle
-implementation "com.xwray:groupie:2.7.0"
+implementation "com.xwray:groupie:$groupie_version"
 ```
 
 Groupie includes a module for Kotlin and Kotlin Android extensions. Never write a ViewHolder again—Kotlin generates view references and Groupie uses a generic holder. [Setup here.](#kotlin) 
 
 ```gradle
-implementation "com.xwray:groupie:2.7.0"
-implementation "com.xwray:groupie-kotlin-android-extensions:2.7.0"
+implementation "com.xwray:groupie:$groupie_version"
+implementation "com.xwray:groupie-kotlin-android-extensions:$groupie_version"
 ```
 
 Groupie also supports Android's [data binding](https://developer.android.com/topic/libraries/data-binding/index.html) to generate view holders. [Setup here.](#data-binding)
 
 ```gradle
-implementation "com.xwray:groupie:2.7.0"
-implementation "com.xwray:groupie-databinding:2.7.0" 
+implementation "com.xwray:groupie:$groupie_version"
+implementation "com.xwray:groupie-databinding:$groupie_version" 
 ```
 
+Groupie also has a support module for Android's [view binding](https://developer.android.com/topic/libraries/view-binding). This module also supports Android data binding, so if your project uses both data vinding and view binding, you don't have to add the dependency on the data binding support module. [Setup here.](#view-binding)
```

Comment:
spelling correction "uses both data vinding and view binding".

Reply:
See, the way it's laid out here basically means that you *no longer* need to use the `data-binding` module if using gradle plugin 3.6.0. This is why I'm saying we should deprecate it soon.

Reply:
OK, now I got your point. Thanks for your patience.